libp2p/NodeTable.h

```diff
@@ -394,7 +401,7 @@ struct DiscoveryDatagram: public RLPXDatagramFace
 
     /// Constructor used for sending.
     DiscoveryDatagram(bi::udp::endpoint const& _to)
-      : RLPXDatagramFace(_to), ts(futureFromEpoch(c_timeToLiveS))
+      : RLPXDatagramFace(_to), timestamp(futureFromEpoch(c_timeToLiveS))
```

Comment:
Since timestamp is the packet expiration time, can we rename it to something like "expiration"?